server/users.ts

```diff
@@ -72,6 +72,16 @@ function move(user: User, newUserid: ID) {
 	user.userid = newUserid;
 	users.set(newUserid, user);
 
+	user.forcedPublic = undefined;
```

Comment:
```suggestion
	user.forcedPublic = null;
```

And here I thought we agreed to use `null` for nullables :p